FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/StandaloneFixtures/SetUp/content.txt

```diff
@@ -0,0 +1,5 @@
+|Import Fixture|
+|dbfit.fixture|
+
+|Database Environment|Informix|
+|Connect|localhost:16466|dbfit|DBFIT|dbfit|
```

Comment:
We might alternatively connect via `Connect Using File`. This is the case with Oracle - in theory it allows easier switch to a custom test database outside the VM.